src/wrapper/repository_wrapper.hpp

```diff
@@ -51,7 +51,8 @@ class repository_wrapper : public wrapper_base<git_repository>
     // Commits
     commit_wrapper find_commit(std::string_view ref_name = "HEAD") const;
     commit_wrapper find_commit(const git_oid& id) const;
-    void create_commit(const signature_wrapper::author_committer_signatures&, const std::string&);
+    void create_commit(const signature_wrapper::author_committer_signatures&, const std::string&, const std::optional<commit_list_wrapper>& parents_list);
```

Comment:
We can use `std::string_view` rather than `const std::string&` for consistency with other string-accepting functions:

```suggestion
    void create_commit(const signature_wrapper::author_committer_signatures&, std::string_view, const std::optional<commit_list_wrapper>& parents_list);
```

The same applies to the line below.